File: src/tool/spring-boot-generator/output/test/test/src/main/java/com/test/pojo/entity/StationInfo.java
/*
 * @Descripttion: 
 * @version: 
 * @Author: fuanlei
 * @Date: 2019-12-17 09:09:58
 * @LastEditors  : fuanlei
 * @LastEditTime : 2019-12-18 14:55:53
 */
package com.undefined.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

@import

@ApiModel("车站信息")
@Getter
@Setter
public class StationInfo {

    @ApiModelProperty("创建时间")
    private Date createTime;

    @ApiModelProperty("车站全称")
    private String fullName;

    @ApiModelProperty("车站编号")
    private Integer id;

    @ApiModelProperty("输入代码")
    private String inputCode;

    @ApiModelProperty("删除标记")
    private Integer isDelete;

    @ApiModelProperty("技术作业别")
    private Integer jobType;

    @ApiModelProperty("操作人")
    private Integer operator;

    @ApiModelProperty("车站简称")
    private String shortName;

    @ApiModelProperty("车站编码")
    private String stationCode;

    @ApiModelProperty("车站等级")
    private Integer stationGrade;

    @ApiModelProperty("车站类型")
    private Integer stationType;

    @ApiModelProperty("状态")
    private Integer status;

    @ApiModelProperty("岔道数量")
    private Integer ternoutNum;

    @ApiModelProperty("股道数量")
    private Integer thighTrackNum;

    @ApiModelProperty("公务段编号")
    private Integer trackDivisionId;

    @ApiModelProperty("更新时间")
    private Date updateTime;

}

File: src/tool/spring-boot-generator/output/railway/db/stationInfo.js
let stationInfo = {
	name: "stationInfo",
	description: "车站信息",
	columns: {
		id: {
			name: "id",
			description: "车站编号",
			isPk: true,
			autoIncrement: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		trackDivisionId: {
			name: "trackDivisionId",
			description: "公务段编号",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		fullName: {
			name: "fullName",
			description: "车站全称",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		shortName: {
			name: "shortName",
			description: "车站简称",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		stationCode: {
			name: "stationCode",
			description: "车站编码",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		inputCode: {
			name: "inputCode",
			description: "输入代码",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		stationType: {
			name: "stationType",
			description: "车站类型",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		stationGrade: {
			name: "stationGrade",
			description: "车站等级",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		jobType: {
			name: "jobType",
			description: "技术作业别",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		thighTrackNum: {
			name: "thighTrackNum",
			description: "股道数量",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		ternoutNum: {
			name: "ternoutNum",
			description: "岔道数量",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		status: {
			name: "status",
			description: "状态",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		isDelete: {
			name: "isDelete",
			description: "删除标记",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		createTime: {
			name: "createTime",
			description: "创建时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		updateTime: {
			name: "updateTime",
			description: "更新时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		operator: {
			name: "operator",
			description: "操作人",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		}
	}
};

let columnsArray = [
	"id",
	"trackDivisionId",
	"fullName",
	"shortName",
	"stationCode",
	"inputCode",
	"stationType",
	"stationGrade",
	"jobType",
	"thighTrackNum",
	"ternoutNum",
	"status",
	"isDelete",
	"createTime",
	"updateTime",
	"operator"
];

function create() {
  return {};
}

stationInfo.columnsArray=columnsArray;
stationInfo.create=create;
module.exports=stationInfo;